src/Command/MailerCitizenProjectValidatedSummaryCommand.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\CitizenProject\CitizenProjectBroadcast;
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\CitizenProject;
+use AppBundle\Repository\AdherentRepository;
+use AppBundle\Repository\CitizenProjectRepository;
+use Doctrine\ORM\EntityManager;
+use Symfony\Bridge\Monolog\Logger;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class MailerCitizenProjectValidatedSummaryCommand extends ContainerAwareCommand
+{
+    public const COMMAND_NAME = 'app:mailer:citizen-project-summary';
+    private const DEFAULT_APPROVED_SINCE = '7 days';
+    private const DEFAULT_OFFSET = 0;
+    private const CITIZEN_PROJECTS_SUMMARY_LIMIT = 15;
+
+    /** @var EntityManager */
+    private $entityManager;
+
+    /** @var Logger */
+    private $logger;
+
+    /** @var CitizenProjectBroadcast */
+    private $broadcast;
+
+    /** @var AdherentRepository */
+    private $adherentRepository;
+
+    /** @var CitizenProjectRepository */
+    private $citizenProjectRepository;
+
+    protected function configure()
+    {
+        $this
+          ->setName(self::COMMAND_NAME)
+          ->addOption(
+              'approvedSince',
```

Comment:
I think the convention for command arguments and options name is to use snake case.